internal/pkg/cli/task_run.go

```diff
@@ -371,6 +379,42 @@ func (o *runTaskOpts) Validate() error {
 		}
 	}
 
+	if len(o.secrets) > 0 {
+		if err := o.validateSecrets(); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (o *runTaskOpts) validateSecrets() error {
+	if o.appName == "" && o.env == "" && !o.acknowledgeSecretsAccess {
+		fmt.Println("Looks like you're requesting ssm:GetParameters to the following SSM parameters:")
+
+		for _, value := range o.secrets {
+			if !arn.IsARN(value) || strings.Contains(value, ":ssm:") {
+				fmt.Println("* " + value)
+			}
+		}
+
+		fmt.Println("\nand secretsmanager:GetSecretValue to the following Secrets Manager secrets:")
+
+		for _, value := range o.secrets {
+			if strings.Contains(value, ":secretsmanager:") {
+				fmt.Println("* " + value)
+			}
+		}
+
+		v, err := o.prompt.Confirm(taskSecretsPermission, "")
+		if err != nil {
+			return fmt.Errorf("failed to confirm deployment: %w", err)
```

Comment:
nit:
```suggestion
			return fmt.Errorf("prompt to confirm secrets access: %w", err)
```
To wrap an error, it might be better if we include contextual information such as what we are trying to do here (i.e. `"prompt to confirm secrets access"`), instead of `"failed to ..."`. In the majority of the codebase we tend to wrap errors this way. My thought is that `"failed to"` is redundant since error has occurred. What would really help here is some contextual information.